scanner/src/routes/style.css

```diff
@@ -0,0 +1,18 @@
+* {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
```

Comment:
`overflow: hidden;` is likely unwanted in general and leads to it being broken in some rare cases where our sizes just don't fit on screen